.github/workflows/integration_tests.yml

```diff
@@ -180,10 +179,9 @@ jobs:
       matrix_os: ${{ steps.export-result.outputs.matrix_os }}
       matrix_ssl: ${{ steps.export-result.outputs.matrix_ssl }}
       apis: ${{ steps.export-result.outputs.apis }}
+      mobile_test_on: ${{ steps.export-result.outputs.mobile_test_on }}
       android_device: ${{ steps.export-result.outputs.android_device }}
-      android_api: ${{ steps.export-result.outputs.android_api }}
```

Comment:
Just making sure we don't need these any more?

Reply:
Yes, android_device contains both android_model & android_api.